src/atomicui/pages/DemoPage/styles.scss

```diff
@@ -1,122 +1,129 @@
 /* Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved. */
 /* SPDX-License-Identifier: MIT-0 */
 
-.maplibregl-canvas-container.mapboxgl-canvas-container {
-	position: relative;
-	z-index: 0;
-}
-
-.amplify-flex.location-disabled {
-	gap: 0;
-	justify-content: center;
-	align-items: center;
-	width: 2.46rem;
-	height: 2.46rem;
-	position: absolute;
-	bottom: 9.85rem;
-	right: 2rem;
-	background-color: var(--white-color);
-	opacity: 0.8;
-	border-radius: 4px;
-	box-shadow: 0px 0px 10px rgb(0 0 0 / 20%);
-	cursor: pointer;
-	z-index: 9;
-
-	@media only screen and (max-width: 1023px) {
-		bottom: 12.5rem;
-	}
-
-	@media only screen and (max-width: 495px) {
-		bottom: 14rem;
+.maplibregl-ctrl-bottom-right {
+	details {
+		right: 1.23rem !important;
+		bottom: 0.25rem !important;
+		font-size: 0.77rem !important;
 	}
 
-	@media only screen and (max-width: 300px) {
-		bottom: 15.5rem;
+	/* Geolocate control */
+	> div:nth-child(2) {
+		justify-content: center !important;
+		align-items: center !important;
+		width: 2.46rem !important;
+		height: 2.46rem !important;
+		border-radius: 0.62rem !important;
+		position: absolute !important;
+
+		.maplibregl-ctrl-geolocate {
+			.maplibregl-ctrl-icon {
+				background-image: url("@demo/assets/svgs/icon-locate-me.svg") !important; // TODO: move to custom assets dir
+
+				@media only screen and (min-width: 1024px) {
+					&:hover {
+						background-image: url("@demo/assets/svgs/icon-locate-me-filled.svg") !important; // TODO: move to custom assets dir
```

Comment:
Are these TODOs happening in a follow-up PR?

Reply:
Having this just for future reference, this doesn't effect the build in any way. This is just for optimization.